### DISCUSSION TITLE
Installation on Windows : Advise on where I am going wrong

### DISCUSSION THREAD

**Discussion by wolvesphil** [Taxel/PlexTraktSync#1094] [Support] [answered]
I currently installed PlexTraktSync on windows, running the script shows that things are being processed, I see my Trakt Watchlist gets updated with new movies so it looks like things are working, however Plex is not syncing it's watched status with Trakt.

What I have noticed:

Kodi episode watch syncs to Trakt using native Trakt plugin
Plex scrobbles a new watch and then marks as watched when complete
PlexTraktSync adds new movies to Trakt Watchlist

What I am trying to achieve is anything watched in Kodi is then marked as watched in Plex, any advise on what may be missing?

I am currently running the script just by using "plextraktsync" command

**Comment by simonc56** [comment]
Did you run the setup.bat to setup a scheduled sync ? or do you manually run the sync every X hours ?

A watched status on trakt will be synced to Plex only if you run the plextracksync script again. That's why it's good to setup a schedule to make it run automatically once a day or twice a day for example.

**Comment by wolvesphil** [comment]
I had setup a sync to run daily.
I have also run a manual sync just and I see no changes to my Plex watched status

**Comment by simonc56** [comment]
Can you show your config file and logs.

**Comment by wolvesphil** [comment]
In the logs I see

ERROR[plexapi]:http://IP:8443: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

If I run setup again is this the best way to re-authenticate?

**Reply by simonc56** [reply]
Yes you can do it to be sure.
Also paste full logs, not only the line you think is relevant.

**Comment by wolvesphil** [comment]
I will do, I wanted to see if a re-install would fix the problem first :)

However I have now run the setup.bat and I see the following when running PlexTraktSync

```python
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2032.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2032.0_x64__qbz5n2kfra8p0\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\kodi\Desktop\PlexTraktSync-0.22.1\plextraktsync\__main__.py", line 16, in <module>
    from plextraktsync.cli import cli
  File "C:\Users\kodi\Desktop\PlexTraktSync-0.22.1\plextraktsync\cli.py", line 4, in <module>
    import click
ModuleNotFoundError: No module named 'click'
```

I see no logs written, my previous version was 0.20 and this is now 0.22 I am trying

What path would the logs be under as the path for 0.20 has nothing in when running now

**Reply by simonc56** [reply]
`ModuleNotFoundError: No module named 'click'`

It seems the setup process wasn't successfull because there is a (many?) missing module.
Please show logs of setup run.

**Reply by wolvesphil** [reply]
Where do I find those logs?

**Reply by simonc56** [reply]
In the command prompt where you run setup.bat
Do not double-clic on setup.bat because it will close as soon as it crashes.
Open [cmd](https://www.howtogeek.com/235101/10-ways-to-open-the-command-prompt-in-windows-10/) yourself and run setup.bat in.

**Comment by wolvesphil** [comment, marked as answer]
When I ran the setup I noticed things didn't seem familiar and then did some digging and realised I installed using pipx on my Windows computer... I have now managed to get my pipx installed version upgraded from 0.20 to 0.22 and things appear to be working now as I see things happening in Plex.

Your advice has helped me resolve this, as it was so long ago I installed this I wouldn't have got to the bottom of it!

I do now appear to have both 0.20 and 0.22 installed, however I have changed my PATH variables to only know of the install path for 0.22 so all appears to be running as it should

**Comment by simonc56** [comment]
Great, you can now edit discussion title to make it accurate, this can help others. And Mark as answer the best comment.
